core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -92,7 +92,7 @@ THE SOFTWARE.
   </j:if>
 </j:if>
 <x:doctype name="html" />
-<html>
+<html class="${h.getSystemProperty('ui.enableNewUi') ? 'new-ui' : ''}">
```

Comment:
we can probably come up with a better name for this, it would have been nice to have included this in the PR description as well 😉 

```suggestion
<html class="${h.getSystemProperty('jenkins.UI_REFRESH') ? 'new-ui' : ''}">
```

```suggestion
<html class="${h.getSystemProperty('jenkins.ui.refresh') ? 'new-ui' : ''}">
```

```suggestion
<html class="${h.getSystemProperty('jenkins.ui.refresh.2020') ? 'new-ui' : ''}">
```